tests/integrations/sqlalchemy/test_sqlalchemy.py

```diff
@@ -189,15 +189,15 @@ def processor(event, hint):
     assert len(json_dumps(event)) < max_bytes
 
     # Some spans are discarded.
-    assert len(event["spans"]) == 999
+    assert len(event["spans"]) == 1000
 
     # Some spans have their descriptions truncated. Because the test always
     # generates the same amount of descriptions and truncation is deterministic,
     # the number here should never change across test runs.
     #
     # Which exact span descriptions are truncated depends on the span durations
     # of each SQL query and is non-deterministic.
-    assert len(event["_meta"]["spans"]) == 536
```

Comment:
Why does this change? If this report changed because of spanrecorder changes I wonder if we should subtract 1 just to not change reporting format

Reply:
I'm not sure why testing for `x - 1 = y` is better than testing `x = y + 1`. We're making a change either way...